src/main/java/com/amazon/janusgraph/graph/AbstractGraph.java

```diff
@@ -0,0 +1,30 @@
+package com.amazon.janusgraph.graph;
+
+import com.amazon.janusgraph.triple.Triple;
+import org.janusgraph.core.JanusGraph;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URL;
+import java.util.Set;
+import java.util.concurrent.*;
+
+
+/**
+ * Created by addisonslabaugh on 6/21/17.
+ */
+public interface AbstractGraph {
```

Comment:
My thought behind this is that every time you want to load a Graph, you need these basic building blocks -- a creation queue, the class loader, the executor service. So creating an abstract class, which new classes can extend, would be useful.

With that in mind, if you still think it's superfluous, I can remove.